lib/mwUtil.js

```diff
@@ -33,6 +33,15 @@ mwUtil.makeETag = (rev, tid, suffix) => {
  * Normalizes the request.params.title and returns it back
  */
 mwUtil.normalizeTitle = (hyper, req, title) =>  mwUtil.getSiteInfo(hyper, req)
+.catch((e) => {
```

Comment:
@gwicke I think it's better to think that the domain should exist, since using this for a non-existent (user-provided) domain is an error. Let's document that the expectation is that the domain exist?